### PR TITLE
School_classes

### PR DESCRIPTION
**Pull request by rasha-saber**

### CODE REVIEW

**Review by Nora-Lootfy — APPROVED** [review]
nice work,, a little hint try to open pull request after your first pushing commit to github

File: app/Http/Controllers/SchoolClassController.php [hunk 1]
```diff
@@ -30,39 +35,30 @@ public function create()
...
-//         'time_to' => 'required|date_format:H:i',
-//     ]);
+    $data=[
+        //'k'= 'v'
+     
+     
+'name' =>$request->name,
+'capacity' =>$request->capacity,
+'is_fulled' =>$request->is_fulled=== 'ON' ? 1 : 0,
+'price' =>($request->price),
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/SchoolClassController.php]: why you add `()` no need for these

File: app/Http/Controllers/SchoolClassController.php [hunk 2]
```diff
@@ -30,39 +35,30 @@ public function create()
...
-       return 'Data added successfully';
-//    $validatedData = $request->validate([
-//         'name' => 'required|string|max:255',
-//         'capacity' => 'required|integer',
-//         'is_fulled' => 'nullable|boolean',
-//         'price' => 'required|numeric',
-//         'time_from' => 'required|date_format:H:i',
-//         'time_to' => 'required|date_format:H:i',
-//     ]);
+    $data=[
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/SchoolClassController.php:42]: try to format your code ya Rasha

**Review by Nora-Lootfy — CHANGES_REQUESTED** [review]
good job ya Rasha

File: app/Http/Controllers/SchoolClassController.php (hunk 2 above)
Comment (rasha-saber) [reply app/Http/Controllers/SchoolClassController.php:42]: اوك تمام شكرا لحضرتك

**Pull request merged**